imports/util/employeeUtil.js

```diff
@@ -0,0 +1,15 @@
+import { removeTask } from "../api/task";
+// eslint-disable-next-line import/prefer-default-export
+export const isAdmin = (id, db) => {
```

Comment:
We already have a isAdmin in authUtil, you can merge these two into one, keep it in either here or authUtil, both location make sense to me. Make sure to use the getId mention in the comment above in the new isAdmin helper to get the employeeId for the admin instead of using accountId to retrieve it.